plugins/org.eclipse.xtext.xbase.lib/src/org/eclipse/xtend2/lib/StringConcatenationClient.java

```diff
@@ -35,6 +35,20 @@
 	protected void appendTo(TargetStringConcatenation target) {
 		throw new UnsupportedOperationException("Clients have to override this.");
 	}
+	
+	/**
+	 * Delegates to {@link #appendTo(TargetStringConcatenation)} to make it publicly accessible.
+	 * 
+	 * @param client
+	 *            the {@link StringConcatenationClient} on which {@link #appendTo(TargetStringConcatenation)} is
+	 *            invoked.
+	 * @param target
+	 *            the {@link TargetStringConcatenation} to-append-to. Never <code>null</code>.
+	 * @since 2.8
+	 */
+	public static void appendToClient(StringConcatenationClient client, TargetStringConcatenation target) {
```

Comment:
The method name suggests that something is appended to the client though its the other way round.